.github/workflows/build.yml

```diff
@@ -39,3 +39,26 @@ jobs:
         - name: Test
           shell: bash
           run: cd build && ctest --output-on-failure
+
+  bz-build:
+      strategy:
+        matrix:
+          os: [ubuntu-latest, windows-latest, macos-latest]
+      runs-on: ${{ matrix.os }}
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Build
+          shell: bash
+          run: |
+            cd "${{ github.workspace }}"
+            bazel build '//:yaml-cpp'
+
+        - name: Test
+          shell: bash
+          run: |
+            cd "${{ github.workspace }}"
+            # For some reason 'bazel test //test:test' gets the first / stripped on windows in CI.
+            # Therefore use this ugly version, which is working on all platforms.
+            bazel test test
```

Comment:
And similarly

`bazel test ...`

Reply:
And this needed to be 'bazel test test'.

'bazel test :all' did not work, because it didn't find any tests.

Reply:
Ok, but that did not work `bazel test ...` reports "no tests found".
`bazel test :all` tries to do something with the imported package repo:http.bzl, which does not apply.

Reply:
Now, how to we proceed? This is the best solution I, not being a bazel expert, can come up with...